### ISSUE TITLE
Name 'hypervisor' is not defined when trying to start script

### ISSUE THREAD

**Issue by SydneyRain** [dhinakg/vm-rpc#1]
Hi!

So, I am having a little problem, when I try to start this python script, it has this error:

```
Traceback (most recent call last):
  File "main.py", line 52, in <module>
    if hypervisor == []:
NameError: name 'hypervisor' is not defined
```

I am on Windows 10 using Python 3.7.4 with VMWare Workstation 15. I did install pypresence but it still doesn't work.

**Comment by SydneyRain** [comment]
Fixed this issue here: https://github.com/dhinakg/vm-rpc/pull/2

**Comment by dhinakg** [comment]
Fixed in #2, thank you!
